Northstar.Custom/mod/scripts/vscripts/_droppod_spawn.gnut

```diff
@@ -35,9 +39,11 @@ void function SpawnPlayersInDropPod( array< entity > players, vector targetOrigi
 	
 	foreach ( entity player in players )
 	{
+		if( !IsValid( player ) )
+			continue
 		if ( !IsAlive( player ) )
 			player.RespawnPlayer( null )
```

Comment:
IsAlive does an internal valid check so in theory you could optimize this a bit by using the internal alive check but it's not necessary
```suggestion
		if( !IsValid( player ) )
			continue
		if ( player.IsEntAlive() )
			player.RespawnPlayer( null )
```